polkadot/xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -119,6 +119,13 @@ pub type DescribeAllTerminal = (
 	DescribeBodyTerminal,
 );
 
+pub type GlobalLocationToAccount<AccountId> = GenericLocationToAccountConverter<
+    AccountId,
+    (GlobalConsensusConvertsFor, GlobalConsensusParachainConvertsFor),
+    (DescribeAllTerminal, DescribeFamily<DescribeAllTerminal>),
+>;
```

Comment:
I'm not sure what this is trying to do, but doesn't look like it would work.

I added [more details to the issue](https://github.com/paritytech/polkadot-sdk/issues/7129#issuecomment-2616410969), hope that helps, please ping me for more details if needed.